clio-dataaccess-model/src/main/scala/org/broadinstitute/clio/server/dataaccess/elasticsearch/ElasticsearchIndex.scala

```diff
@@ -132,16 +133,18 @@ object ElasticsearchIndex extends ModelAutoDerivation {
       // Despite being decoupled from "v1", we append -v2 to keep ES indices consistent with GCS.
       // Since we compute GCS paths from the ES index name, inconsistency would break GCS paths.
       indexName = "gvcf-v2",
-      ElasticsearchFieldMapper.StringsToTextFieldsWithSubKeywords
+      ElasticsearchFieldMapper.StringsToTextFieldsWithSubKeywords,
+      Json.obj("data_type" -> DataType.WGS.asJson)
```

Comment:
Any way to avoid the magic string here? Or at least store it in a constant, as we do with `UpsertIdElasticsearchName` and `EntityIdElasticsearchName`.